cmake/templates/BoostConfig.cmake.in

```diff
@@ -103,10 +103,22 @@ if(TARGET Boost::iostreams)
   include("${CMAKE_CURRENT_LIST_DIR}/BoostBZip2Dependency.cmake" OPTIONAL)
 endif()
 
+include(CheckLibraryExists) # check_library_exists
+check_library_exists(rt clock_gettime "" _boost_check_rt_library)
+
 if(TARGET Boost::thread)
   find_package(Threads REQUIRED)
   set_property(
     TARGET "Boost::thread"
-    APPEND PROPERTY INTERFACE_LINK_LIBRARIES Threads::Threads
-  )
+    APPEND PROPERTY INTERFACE_LINK_LIBRARIES Threads::Threads rt
```

Comment:
Separate component - separate testing - separate problems. This change should appear in previous pull request. Let's focus on chrono now.